lib/Data/Database.php

```diff
@@ -542,7 +548,15 @@ protected function _getExpiredPastes($batchsize)
     private static function _exec($sql, array $params)
     {
         $statement = self::$_db->prepare($sql);
-        $result    = $statement->execute($params);
+        foreach ($params as $key => $parameter) {
+            $position = $key + 1;
+            if (is_int($parameter)) {
+                $statement->bindValue($position, $parameter, PDO::PARAM_INT);
+            } else {
+                $statement->bindValue($position, $parameter);
+            }
+        }
```

Comment:
```suggestion
        foreach ($params as $key => &$parameter) {
            if (is_int($parameter)) {
                $statement->bindParam($key + 1, $parameter, PDO::PARAM_INT);
            } elseif (strlen($parameter) >= 4000) {
                $statement->bindParam($key + 1, $parameter, PDO::PARAM_STR, strlen($parameter));
            } else {
                $statement->bindParam($key + 1, $parameter);
            }
        }
```
Seems like inserting CLOB absolutely require a length argument (I apologize for not testing thoroughly last time), which is only available in `bindParam`. But then `bindParam` is by reference...

Reply:
No worries, and thanks for re-testing.